embed-code-go/test/embedding_test/embedding_test.go

```diff
@@ -97,16 +98,15 @@ func (suite *EmbeddingTestSuite) TestFalseTransitions() {
 	}
 
 	falseProcessor := embedding.NewEmbeddingProcessorWithTransitions(docPath, suite.config, falseTransitions)
-
-	suite.Require().Panics(func() {
-		falseProcessor.Embed()
-	})
+	err := falseProcessor.Embed()
+	suite.Require().Error(err, "There must be an error during the embedding with the false transitions.")
```

Comment:
You say not what you wanted to say. The error was expected, but there was none.

"There must be" means "something went wrong, most likely, there was an error".